docs/user/precheck/check_networks.rst

```diff
@@ -0,0 +1,199 @@
+A traditional L2 ethernet network is a ``flat`` network. Any servers attached
+to this network are able to see the same broadcast traffic and can contact each
+other without requiring a router. ``flat`` networks are often used to attach
+Nova servers to an existing L2 network (this is called a ``provider network``).
+
+``physical_network`` parameter is required for creating ``flat`` network in
+Neutron. And this physical network can be used only once for any ``flat``
+network. So, if any ``flat`` network already exists on the cloud, there is no
+possibility to create another ``flat`` network with the same physical
+network. And this check exists for determining such sorts of busyness.
+
+
+Physical networks existence on DST for VLAN network type
+--------------------------------------------------------
+
+A ``vlan`` network is one that uses VLANs for segmentation. When user creates a
+new network in Neutron, it will be assigned a VLAN ID from the range user has
+configured in his Neutron configuration. Using ``vlan`` networks requires that
+any switches in user's environment are configured to trunk the corresponding
+VLANs.
+
+``physical_network`` parameter is required for creating ``vlan`` network in
+Neutron. And it also should be defined in the Neutron configuration along with
+the VLAN IDs range.  Otherwise the error will occur during the network
+migration::
+
+    Invalid input for operation: physical_network '<physnet-name>' unknown  for
+    VLAN provider network. (HTTP 400)
+
+This check verifies if physical network from the source cloud's network also
+exists on the destination cloud too.
+
+
+.. note::
+
+    Physical networks are defined in the Neutron configuration (on the server
+    side). For example:
+
+    If Neutron ML2 plugin is used on the cloud, physical networks for ``flat``
+    and ``vlan`` networks will be defined in the
+    :file:`/etc/neutron/plugins/ml2/ml2_conf.ini`::
+
+        [ml2]
+        # (ListOpt) List of network type driver entrypoints to be loaded from
+        # the neutron.ml2.type_drivers namespace.
+        #
+        type_drivers = vlan,flat
+
+        [ml2_type_flat]
+        # (ListOpt) List of physical_network names with which flat networks
+        # can be created. Use * to allow flat networks with arbitrary
+        # physical_network names.
+        #
+        flat_networks = physnet1,physnet2
+
+        [ml2_type_vlan]
+        # (ListOpt) List of <physical_network>[:<vlan_min>:<vlan_max>] tuples
+        # specifying physical_network names usable for VLAN provider and
+        # tenant networks, as well as ranges of VLAN tags on each
+        # physical_network available for allocation as tenant networks.
+        #
+        network_vlan_ranges = physnet3:300:319,physnet4:400:419,physnet5
+
+    In case of above error occurs:
+     - Compare these config files on source and destination clouds;
+     - Add missing physical networks;
+     - Reconfigure Neutron to work with these physnets.
```

Comment:
Similar to sections above I'd suggest that you split this into:

 - Overview of VLAN networks
 - What this check does
 - Why this check is important and what kinds of failures it can lead to if failed